lib/routes/mangadex/mdlist/feed.ts

```diff
@@ -0,0 +1,161 @@
+                optional: true,
+            },
+            {
+                name: 'MANGADEX_REFRESH_TOKEN',
+                description: 'MangaDex Refresh Token, required when username and password are not set and the list is private',
+                optional: true,
+            },
+            {
+                name: 'MANGADEX_FILTERED_LANGUAGES',
+                description: 'MangaDex Filtered Languages',
+                optional: true,
+            },
+        ],
+    },
+    handler,
+};
+
+async function handler(ctx) {
+    const { id, lang } = ctx.req.param();
+
+    const limit = ctx.req.query('limit') ? Number.parseInt(ctx.req.query('limit')) : DEFAULT_LIMIT;
+    const isPrivate = !!ctx.req.query('private');
+
+    const accessToken = isPrivate ? await getToken() : undefined;
+
+    const languagesQuery = new Set([...(typeof lang === 'string' ? [lang] : lang || []), ...(await getFilteredLanguages())].filter(Boolean));
+
+    const { listName, listAuthor } = (await cache.tryGet(
+        `mangadex:mdlist-info-${id}`,
+        async () => {
+            const response = await got.get(
+                `${constants.API.BASE}/list/${id}${toQueryString({
+                    includes: ['user'],
+                })}`,
+                {
+                    headers: {
+                        Authorization: String(isPrivate ? `Bearer ${accessToken}` : ''),
+                        'User-Agent': config.trueUA,
+                    },
+                }
+            );
+
+            const mdlistInfo = response?.data?.data;
+            if (!mdlistInfo) {
+                throw new Error('Failed to retrieve user follows from MangaDex API.');
+            }
+
+            const listName = mdlistInfo.attributes.name;
+            const listAuthor = mdlistInfo.relationships.find((relationship) => relationship.type === 'user')?.attributes.username;
+
+            return { listName, listAuthor };
+        },
+        config.cache.contentExpire
+    )) as Record<string, any>;
+
+    const feed = (await cache.tryGet(
+        `mangadex:mdlist-feed-${id}`,
+        async () => {
+            const response = await got.get(
+                `${constants.API.BASE}/list/${id}/feed${toQueryString({
+                    limit,
+                    translatedLanguage: languagesQuery,
+                    order: {
+                        publishAt: 'desc',
+                    },
+                })}`,
+                {
+                    headers: {
+                        Authorization: String(isPrivate ? `Bearer ${accessToken}` : ''),
+                        'User-Agent': config.trueUA,
+                    },
+                }
+            );
+
+            const feed = response?.data?.data;
+            if (!feed) {
+                throw new Error('Failed to retrieve user follows from MangaDex API.');
+            }
+
+            return feed;
+        },
+        config.cache.contentExpire,
```

Comment:
https://github.com/DIYgod/RSSHub/pull/18269#discussion_r2072443949
```suggestion
        config.cache.routeExpire,
```
This belongs to a hot cache which requires a shorter TTL.